app/_hub/kong-inc/jwt-signer/index.md

```diff
@@ -639,12 +639,12 @@ params:
       default: false
       datatype: boolean
       description: |
-        In case when channel token is not provided or no `config.channel_token_request_header` is specified,
-        the plugin cannot obviously verify the channel token. In that case the plugin normally responds
+        If a channel token is not provided or no `config.channel_token_request_header` is specified,
+        the plugin cannot obviously verify the channel token. In that case, the plugin normally responds
         with `401 Unauthorized` (client didn't send a token) or `500 Unexpected` (a configuration error).
-        With this parameter you can let the request to proceed even when there is no token to be checked.
-        If the token is provided, then this parameter has no effect (look other parameters to enable and
-        disable checks in that case).
+        Enable this parameter to allow the request to proceed even when there is no channel token
+        to check. If the channel token is provided, then this parameter has no effect
+        (look other parameters to enable and disable checks in that case).
```

Comment:
@bungle I'm not sure what the intent of this phrase is?

> (look other parameters to enable and disable checks in that case)

Can you provide additional information? Or should it just be removed?